specs/byron_minimal.cddl

```diff
@@ -0,0 +1,60 @@
+; Basic Cardano Types
+
+; blake2b224 = bytes .size 28
+; public_ed_25519_bip32 = bytes .size 64
+; script = bytes .size 32
+; public_ed_25519 = bytes .size 32
+
+blake2b224 = _CDDL_CODEGEN_RAW_BYTES_TYPE_
+public_ed_25519_bip32 = _CDDL_CODEGEN_RAW_BYTES_TYPE_
+byron_script = _CDDL_CODEGEN_RAW_BYTES_TYPE_
+public_ed_25519 = _CDDL_CODEGEN_RAW_BYTES_TYPE_
+
+; blake2b224 of sha256 of cbor(byron_addr_type, spending_data, addr_attributes)
+address_id = _CDDL_CODEGEN_RAW_BYTES_TYPE_ ; blake2b224 
+stakeholder_id = _CDDL_CODEGEN_RAW_BYTES_TYPE_; blake2b224
+crc32 = _CDDL_CODEGEN_EXTERN_TYPE_ ; uint .size 4 ; @newtype
+protocol_magic = _CDDL_CODEGEN_EXTERN_TYPE_ ; uint .size 4 ; @newtype
+hd_address_payload = bytes ; @newtype
+
+; Addresses
+
+; cddl had bootstrap as (1, uint) but byron/mod.rs had no uint field.
```

Comment:
don't remember off the top of my head the history to this, but I can look into it to try and remember

Reply:
I think this may have just been a mistake on my part. The original cddl is `addrdistr = [1] / [0, stakeholderid]` and I think maybe I added a `uint` for the `[1]` case even though it shouldn't have been there